armi/materials/mgO.py

```diff
@@ -30,8 +30,8 @@ class MgO(Material):
     def __init__(self):
         Material.__init__(self)
         """same reference as linear expansion. Table II.
-        Reference density is from Wolfram Alpha At STP (273 K)"""
-
+        Reference density is from Wolfram Alpha At STP (273 K)
+        """
```

Comment:
while editing, might as well bring up to formatting standards. 

Single line starting with capital letter, ending with `.`, blank line, additional info.

Reply:
fixed